jxl/src/headers/frame_header.rs

```diff
@@ -417,6 +420,26 @@ pub struct FrameHeader {
     #[default(if frame_height == 0 { nonserialized.img_height } else { frame_height })]
     #[condition(false)]
     pub height: u32,
+
+    #[coder(Bits(0))]
+    #[default({
```

Comment:
Make that into a free function? It's a fair amount of code for an attribute, and effectively duplicated

Reply:
like this?